validator/client/src/main/java/tech/pegasys/teku/validator/client/ValidatorClientService.java

```diff
@@ -410,7 +410,8 @@ private void scheduleValidatorsDuties(
             forkProvider,
             validatorApiChannel,
             blockContainerSigner,
-            config.getValidatorConfig().isBlindedBeaconBlocksEnabled(),
+            config.getValidatorConfig().isFailoversPublishSignedDutiesEnabled(),
```

Comment:
shouldn't it be `isBlindedBeaconBlocksEnabled()`?

Reply:
My bad! Fixed it